.circleci/config.yml

```
@@ -219,6 +223,18 @@ jobs:
           name: Shellcheck Lint
           command: yarn lint:shellcheck
 
+  test-integration-storybook:
+    docker:
+      - image: koalaman/shellcheck-alpine@sha256:169a51b086af0ab181e32801c15deb78944bb433d4f2c0a21cc30d4e60547065
```

Was the `shellcheck` image used intentionally? I would expect we'd want the node image, since `storybook` is being run in this step
```suggestion
      - image: circleci/node@sha256:e16740707de2ebed45c05d507f33ef204902349c7356d720610b5ec6a35d3d88
```